frontend/src/ui-components/components/Button.vue

```diff
@@ -10,8 +18,15 @@
         <span v-if="hasIconRight" class="ff-btn--icon ff-btn--icon-right">
             <slot name="icon-right"></slot>
         </span>
-    </a>
-    <button v-else ref="input" class="ff-btn transition-fade--color" :type="type" :class="'ff-btn--' + kind + (hasIcon ? ' ff-btn-icon' : '') + (size === 'small' ? ' ff-btn-small' : '') + (size === 'full-width' ? ' ff-btn-fwidth' : '')" :disabled="disabled===true?'true':null" @click="go()">
+    </router-link>
+
+    <button v-else ref="input"
+            class="ff-btn transition-fade--color"
+            :type="type"
+            :class="computedClass"
+            :disabled="htmlDisabled"
+            @mouseup="go"
```

Comment:
Why `mouseup` and not `click`?

Reply:
vue's '@click' handler doesn't get triggered on mouse 3 (middle mouse button) clicks, this is the way to do it